docs/vscode-extension/envchecker-help.md

```diff
@@ -149,6 +149,13 @@ The configurations in *.fx/configs/config.local.json* are like:
 
 Note that the `botEndpoint` should use https protocol.
 
+## Sideloading Permission Disabled
+Your Microsoft 365 tenant admin hasn't enabled sideloading permission for your account. You can't install your app to Teams! 
+
+### Solutions: 
+1. Contact your tenant administrator to resolve this issue. Click this [document](https://docs.microsoft.com/en-us/microsoftteams/platform/toolkit/accounts#teams-custom-app-upload-or-sideload-permission) to learn more about Sideloading.
```

Comment:
"Contact your M365 tenant administrator to turn on sideloading permission for your tenant."

Reply:
and the doc link should be (https://docs.microsoft.com/en-us/microsoftteams/platform/toolkit/accounts#teams-custom-app-upload-or-sideload-permission)